webrob/neems/neem.py

```diff
@@ -61,9 +76,17 @@ def get_info(self):
 
     def activate(self):
         app.logger.info('Activate neem')
-        if not USE_HOST_KNOWROB and not container_started(current_user.username):
+        try:
+            neemHubSettings = get_settings()
+        except SQLAlchemyError as e:
```

Comment:
why not adding a flask handler for this error?

Reply:
done. Now SQLAlchemyError has its own flask handler.